playwright/integration/landing.spec.ts

```diff
@@ -28,6 +28,6 @@ test("Shows error boundary for invalid collection", async ({ page, port }) => {
   await page.goto(`http://localhost:${port}${INVALID_COLLECTION_PATH}`);
 
   const alert = page.getByRole("alert");
-  const errorText = alert.getByText("An error 500 occurred on server");
+  const errorText = alert.getByText("Something went wrong on our end");
```

Comment:
Are there updated designs? 

In the draft design, I see "Oops! Something went wrong while loading this page."

Reply:
The updated designs are under V2 - FINAL